controllers/tasksRequests.js

```diff
@@ -0,0 +1,60 @@
+const taskRequestsModel = require("../models/taskRequests");
+
+const fetchTaskRequests = async (_, res) => {
+  try {
+    const taskRequests = await taskRequestsModel.fetchTaskRequests();
+
+    if (taskRequests && taskRequests.length > 0) {
```

Comment:
You won't need to add this check, if the response array is empty then it's still correct.